SQF/dayz_code/compile/A_Plot_for_Life/fn_check_owner_friends.sqf

```diff
@@ -42,14 +42,17 @@ if (DZE_permanentPlot) then {
 	
 	_playerUID = [_player] call FNC_GetPlayerUID;
 	_targetOwner = _target getVariable ["ownerPUID","0"];
-	_owner = (_playerUID == _targetOwner);
 } else {
+	_playerUID = dayz_characterID;
 	_friendlies	= _player getVariable ["friendlyTo",[]];
 	_targetOwner = _target getVariable ["CharacterID","0"];
-	_owner = (_targetOwner == dayz_characterID);
 };
 
-if (_targetOwner in _friendlies) then {
+// Check if player is owner of target object
```

Comment:
For DZE_permanentPlot = false; it should be checking if _targetOwner is in _friendlies. Your code is checking if dayz_characterID is in _friendlies in that case.